packages/swapper/src/swappers/thorchain/utils/estimateTradeFee/estimateTradeFee.ts

```diff
@@ -5,12 +5,17 @@ import { SwapError, SwapErrorTypes } from '../../../../api'
 import { InboundResponse, ThorchainSwapperDeps } from '../../types'
 import { THOR_TRADE_FEE_MULTIPLIERS } from '../constants'
 import { getPriceRatio } from '../getPriceRatio/getPriceRatio'
+import { isRune } from '../isRune/isRune'
 import { thorService } from '../thorService'
 
+// TODO rename this estimateOutboundFee or similar
 export const estimateTradeFee = async (
   deps: ThorchainSwapperDeps,
   buyAsset: Asset,
 ): Promise<string> => {
+  if (isRune(buyAsset.assetId)) {
+    return '0.02' // todo - read from TC mimir/constants
```

Comment:
@asamere Do we want to expose `NativeTransactionFee` (https://thornode.ninerealms.com/thorchain/constants) in unchained, or is the plan to query a thor node endpoint directly and pass this one over?